core/services/ocr2/plugins/ocr2keeper/evm21/streams_lookup.go

```diff
@@ -59,6 +59,13 @@ type MercuryV02Response struct {
 // MercuryV03Response represents a JSON structure used by Mercury v0.3
 type MercuryV03Response struct {
 	Reports []MercuryV03Report `json:"reports"`
+	Errors  []ErrorItem        `json:"errors"`
```

Comment:
did mercury change their response struct? Is this optional field?

I do not see this being used in prod code?

Reply:
removing for now